pkg/types/core/relayerset.go

```diff
@@ -0,0 +1,30 @@
+package core
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+)
+
+type RelayerSet interface {
+	Get(ctx context.Context, relayID types.RelayID) (Relayer, error)
+	GetAll(ctx context.Context) ([]Relayer, error)
```

Comment:
prefer List to GetAll. 

additionally if you want to complete, you can pass a vardiac filter `List(ctx, ids ...types.RelayID)` and return everything when len(ids) == 0